### PR TITLE
CONCF-78 add delay before showing spinner on page loading

### PR DESCRIPTION
**Pull request by RafLeszczynski**
PLEASE DO NOT MERGE

### CODE REVIEW

File: front/scripts/main/controllers/ApplicationController.ts [hunk 1]
```diff
@@ -28,10 +30,13 @@ App.ApplicationController = Em.Controller.extend({
...
 
 	showLoader: function () {
-		this.set('isLoading', true);
+		this.set('spinnerTimeout', setTimeout(() => {
+			this.set('isLoading', true)
+		}, this.get('spinnerDelay')));
 	},
 
 	hideLoader: function () {
+		clearTimeout(this.get('spinnerTimeout'));
```
Comment (rogatty) [inline front/scripts/main/controllers/ApplicationController.ts]: Look at http://emberjs.com/api/classes/Ember.run.html#method_cancel

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
Hmm ok this delays showing the spinner, but the spinner itself was to rotate faster :(

**Comment by rogatty** [comment]
LGTM

**Pull request merged**